### DISCUSSION TITLE
500+ active installs in Google Play :tada:

### DISCUSSION THREAD

**Discussion by syt0r** [syt0r/Kanji-Dojo#14] [Announcements]
Recently app received huge boost in installs so I decided to share this with community (if there is someone here, lol)

Here is report from Google Play console with installs count since app release:
![image](https://user-images.githubusercontent.com/7478653/222805873-9ded93d9-441b-4478-80b9-5843f2d28c8a.png)

**Comment by Audman** [comment]
I started to learn kanji around a month ago, after talking to a japanese guy for the first time. 
I searched in fDroid "Kanji" and this app was one of the firsts in the matches. Now I know around 150-200 kanjis.

Maybe it's fate 😁

Thank you for this amazing app!

**Comment by argr1971** [comment]
Hallo
die beste derartige App. Das suche ich schon lange.
Das Problem: Ich möchte nicht Japanisch lernen sondern Chinesisch, Kurzzeichen und die sind nicht ganz gleich.
Könnt ihr das in der App einfügen?
